extensions.json

```diff
@@ -461,7 +461,7 @@
       "repository": "https://github.com/JustusAdam/language-haskell",
       "version": "3.3.0",
       "checkout": "v3.3.0",
-      "prepublish": "npm i --save-dev @types/node@10"
+      "prepublish": "npm i --save-dev @types/node@10 && make all"
```

Comment:
I tested this locally and worked - and it seems like the `ubuntu-latest` image contains `build-essentials` package which has `make`. But I wasn't able to test it in the GH workflow environment. 

Would be cool if there was a 'dry run' for PR validation! 😄